app/views/admin/assets/index.html.erb

```diff
@@ -39,9 +39,10 @@
               <div>N comments / N listens</div>
             </div>
             <div class="button_column">
-              <%= button_to "Mark As Spam", spam_admin_asset_path(asset.id), method: :put %>
-              <%= button_to "Spam All for IP", mark_group_as_spam_admin_assets_path, method: :put, params: { mark_spam_by: { user_id: asset.user_id } } %>
-              <%= button_to "Delete User", admin_user_path(asset.user_id), data: { confirm: 'This will delete user forevAr. Are you sure?'}, method: :delete %>
+              <%= button_to "Mark As Spam", spam_admin_asset_path(asset.id), method: :put, remote: true, data: { target: 'admin--asset.spamButton', action: 'click->admin--asset#showUnspamButton'} %>
+              <%= button_to "Mark As Ham", unspam_admin_asset_path(asset.id), method: :put, remote: true, data: { target: 'admin--asset.unspamButton', action: 'click->admin--asset#showSpamButton'} %>
+              <%#= button_to "Spam All for IP", mark_group_as_spam_admin_assets_path, method: :put, params: { mark_spam_by: { user_id: asset.user_id } } %>
```

Comment:
Don't think we necessarily care about this functionality right now, commenting out for now (rather than delete) so we keep it in our field of vision.

Reply:
I think we can remove this line. I ended up implementing this feature on users controller, that's where it was intended, I believe. This was copied and pasted from there.